src/accounts/EthereumAccount.js

```diff
@@ -9,16 +9,12 @@ import { getWalletTitle, getWalletConfig } from '../wallet'
 import SimpleMultiSigContractArtifacts from '../contracts/SimpleMultiSig.json'
 import WalletUtils from '../wallets/utils'
 import env from '../typedEnv'
-import { erc20TokensList } from '../erc20Tokens'
 const PLATFORM_TYPE = 'ethereum'
 
 export default class EthereumAccount implements IAccount<AccountData> {
   accountData: AccountData
 
   constructor (accountData: AccountData) {
-    if (!['ethereum', ...erc20TokensList].includes(accountData.cryptoType)) {
```

Comment:
Making the check here would make this class component rely on the getAllEthContract request. If that the getAllEthContract request failed, this part would fail too.